oneview_redfish_toolkit/blueprints/chassis.py

```diff
@@ -45,7 +46,13 @@ def get_chassis(uuid):
             JSON: JSON with Chassis.
     """
     try:
-        oneview_client = util.get_oneview_client()
+        if util.config["redfish"]["authentication_mode"] == "session":
+            # Revocer session id
```

Comment:
**Recover** not **Revocer**. Fix that in the other blueprints.

Reply:
fixed